haiku-apps/functionplotter/functionplotter-0.9.2.recipe

```diff
@@ -4,13 +4,15 @@ to a printer and can be copied to the clipboard.
 
 Includes a small functional programming language to define constants and \
 functions."
-HOMEPAGE="https://github.com/haikuports/haikuports/"
+HOMEPAGE="https://github.com/HaikuArchives/FunctionPlotter"
 COPYRIGHT="2000 Michael Pfeiffer"
 LICENSE="GNU GPL v2"
 REVISION="1"
-SOURCE_URI="https://github.com/haikuports/haikuports/files/2381969/FunctionPlotter0.9.2.src.zip"
-CHECKSUM_SHA256="07795a291a3c756e89a1d49d43ea2d2628e9728d38c8aaeb96ee1d70e7fae548"
-SOURCE_DIR="FunctionPlotter0.9.2"
+srcGitRev="31c26ada63f9ea31cf8ef0feb7224ce355f31631"
+SOURCE_URI="https://github.com/HaikuArchives/FunctionPlotter/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="d2345daf976c87399d5d487dd2f2a374a4ff0988c97bbeed6abf5dd6fa764266"
+SOURCE_FILENAME="FunctionPlotter-$portVersion.tar.gz"
+SOURCE_DIR="FunctionPlotter-$srcGitRev"
 ADDITIONAL_FILES="Makefile"
```

Comment:
Suggestions: include `-$srcGitRev` in `SOURCE_FILENAME` and move the Makefile to [upstream](https://github.com/HaikuArchives/FunctionPlotter/tree/master/src), as you have considered before the repo was created.
```diff
-SOURCE_FILENAME="FunctionPlotter-$portVersion.tar.gz"
+SOURCE_FILENAME="FunctionPlotter-$portVersion-$srcGitRev.tar.gz"
 SOURCE_DIR="FunctionPlotter-$srcGitRev"
-ADDITIONAL_FILES="Makefile"
```
```diff
 BUILD()
 {
-	cp "$portDir"/additional-files/Makefile src
 	make $jobArgs -C src
 }
```

Reply:
PR created for the Makefile: https://github.com/HaikuArchives/FunctionPlotter/pull/1
Once that's integrated I'll bump this recipe with the changes :)